corehq/apps/userreports/sql/adapter.py

```diff
@@ -31,6 +32,12 @@ def drop_table(self):
         with self.engine.begin() as connection:
             self.get_table().drop(connection, checkfirst=True)
 
+    def get_query_object(self):
+        """
+        Get a sqlalchemy query object ready to query this table
+        """
+        return self.sesssion_factory.Session.query(self.get_table())
```

Comment:
I think `.query` should be called on an actual `Session` object and not the session maker so this should be `self.sesssion_factory.Session().query(self.get_table())`